server/models/follow.js

```diff
@@ -0,0 +1,21 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const FollowSchema = new Schema({
+    userId: {
+        type: String,
+        ref: 'User',
+    },
```

Comment:
If this is User._id it's not a String, it's a Schema.Types.ObjectID.  If it's the username that might be useful, and I'm not totally opposed to including it, BUT it'll require more work with any update.  If you decide to include username you need to 1) change the field's name to username and 2) add the real User._id.